tests/query/v2/test_get_subgraph.py

```diff
@@ -0,0 +1,987 @@
+# --coding:utf-8--
+#
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+from tests.common.nebula_test_suite import NebulaTestSuite
+from tests.common.nebula_test_suite import T_EMPTY, T_NULL
+import pytest
+
+class TestSubGraph(NebulaTestSuite):
+    @classmethod
+    def prepare(self):
+        self.use_nba()
+        self.load_vertex_edge()
+
+    def cleanup():
+        pass
+
+    def test_invalid_input(self):
+        stmt = 'GET SUBGRAPH FROM $-.id'
+        resp = self.execute_query(stmt)
+        self.check_resp_failed(resp)
+
+        stmt = 'GET SUBGRAPH FROM $a.id'
+        resp = self.execute_query(stmt)
+        self.check_resp_failed(resp)
+
+        stmt = 'GO FROM "Tim Duncan" OVER like YIELD $$.player.age AS id | GET SUBGRAPH FROM $-.id'
+        resp = self.execute_query(stmt)
+        self.check_resp_failed(resp)
+
+        stmt = '$a = GO FROM "Tim Duncan" OVER like YIELD $$.player.age AS ID; GET SUBGRAPH FROM $a.ID'
+        resp = self.execute_query(stmt)
+        self.check_resp_failed(resp)
+
+        stmt = '$a = GO FROM "Tim Duncan" OVER like YIELD like._src AS src; GET SUBGRAPH FROM $b.src'
+        resp = self.execute_query(stmt)
+        self.check_resp_failed(resp)
+
+        # stmt = 'GO FROM "Tim Duncan" OVER like YIELD $$._dst AS id, $^._src AS id | GET SUBGRAPH FROM $-.id'
```

Comment:
maybe want to test duplicate colname, modify to `YIELD like._dst AS id, like._src AS id`